requirements.txt

```diff
@@ -8,5 +8,5 @@ PyYAML>=5.1, !=5.4.*  # OmegaConf requirement >=5.1
 tqdm>=4.41.0
 fsspec[http]>=0.8.1
 tensorboard>=2.2.0
-torchmetrics>=0.2.0
+torchmetrics>=0.3.0
```

Comment:
we could just pin == to avoid it in the future

Reply:
that was my first thought too but that would block any user using newer TM

Reply:
don't we want users to import torchmetrics directly?

Reply:
yes but still we need to handle compatibility